package.json

```diff
@@ -20,6 +20,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.51.1",
+    "react-hot-toast": "^2.4.1",
```

Comment:
We've deleted this lib currently

Reply:
react-hot-toast and its component in layout removed